### DISCUSSION TITLE
Package for pre-built binaries

### DISCUSSION THREAD

**Discussion by Jeordanis** [microsoft/vcpkg#41430] [Q&A] [answered]
Is it possible to create a package from pre-built binaries in a way that cmake’s find_package can find it? 

I packaged some binaries using:
```cmake
file(INSTALL ${CMAKE_CURRENT_LIST_DIR}/lib DESTINATION ${CURRENT_PACKAGES_DIR})
file(INSTALL ${CMAKE_CURRENT_LIST_DIR}/include DESTINATION ${CURRENT_PACKAGES_DIR})
```
However, this does not generate the cmake config files needed for “find_packake” to work. 

Do I have to create the config files manually or is there a way that I can make vcpkg generate them?

**Comment by autoantwort** [comment]
You have to generate them manually

**Comment by autoantwort** [comment, marked as answer]
You have to generate them manually. As an example you could use: https://github.com/autoantwort/Lichtsteuerung/tree/master/overlay-ports/susbdmx

**Reply by Jeordanis** [reply]
Thank you! It works.

I was hopping for vcpkg to generate those automatically. Making them manually is complex. Specially for a multi-platform project.
